cmd/output/package_test.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package output_test
```

Comment:
Prefer putting _package_test.go_ files in the logic's package (`output` in this case).